internal/status/ing_status.go

```diff
@@ -266,7 +266,12 @@ func patchIngressAnnotations(ingObj *networkingv1beta1.Ingress, vsAnnotations ma
 	return nil
 }
 
-func DeleteIngressStatus(svc_mdata_obj avicache.ServiceMetadataObj, isVSDelete bool, key string) error {
+func DeleteIngressStatus(options []UpdateOptions, isVSDelete bool, key string) error {
+	//func DeleteIngressStatus(svc_mdata_obj avicache.ServiceMetadataObj, isVSDelete bool, key string) error {
```

Comment:
stale comment. Also when would the options length be 0?

Reply:
Removed. The length should not be zero, though I think it would be always one now. I'll see if we can remove the slice in a PR later, don't want to reformat that now.